WordPress/gutenberg
# Adding a GIF badge to image blocks with gifs in them

## Description
Fixes [#968](https://github.com/wordpress-mobile/gutenberg-mobile/issues/968)
Adds a GIF badge to image blocks with a GIF loaded.

## Testing Instructions
1. Create a post
2. Add an image block
3. Upload a gif to the image block
4. See that the "GIF" badge on top left appears.
5. Replace the gif with a regular photo.
6. See that there is no badge.

## Screenshots 
<table width="100%">
  <tr><th>iOS</th><th>Android</th></tr>
  <tr>
  <td width="50%"><video src="https://user-images.githubusercontent.com/13263478/155195570-45901d5d-8492-4bca-829d-4c6b8d6714c7.mp4" /></td>
  <td width="50%"><img src="https://user-images.githubusercontent.com/13263478/155195307-b8d7b15f-1c32-4b88-aff5-3dd5773f1df9.png" /></td>
  </tr>
</table>

## Types of changes
New feature

## Checklist:
